src/main/java/com/example/restapi/entities/Employee.java

```diff
@@ -0,0 +1,41 @@
+package com.example.restapi.entities;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import java.io.Serializable;
+import java.util.Date;
+
+
+@Entity
+@Table
+@Data
+@NoArgsConstructor
+public class Employee implements Serializable {
+	private static final long serialVersionUID = -297553281792804396L;
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	private Long id;
+
+	@Column(name = "firstname")
```

Comment:
column name should be snake case (underscores separate words)

Reply:
Fixed